spark/components/icons/react/SprkIcon.stories.js

```diff
@@ -7,18 +7,374 @@ export default {
+
+export const plusFilledSmall = () => <SprkIcon iconName="plus-filled-small" additionalClasses="sprk-c-Icon--filled" />;
+
+export const prequalified = () => <SprkIcon iconName="prequalified" additionalClasses="sprk-c-Icon--xl" />;
+
+export const prequalifiedFilled = () => <SprkIcon iconName="prequalified-filled" additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled" />;
+
+export const prequalifiedFilledSmall = () => <SprkIcon iconName="prequalified-filled-small" additionalClasses="sprk-c-Icon--filled" />;
+
+export const pricePerSqFt = () => <SprkIcon iconName="price-per-sq-ft" additionalClasses="sprk-c-Icon--xl" />;
+
+export const print = () => <SprkIcon iconName="print" additionalClasses="sprk-c-Icon--xl" />;
+
+export const question = () => <SprkIcon iconName="question" additionalClasses="sprk-c-Icon--xl" />;
+
+export const questionFilled = () => <SprkIcon iconName="question-filled" additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled" />;
+
+export const questionFilledSmall = () => <SprkIcon iconName="question-filled-small" additionalClasses="sprk-c-Icon--filled" />;
+
+export const rateShield = () => <SprkIcon iconName="rate-shield" additionalClasses="sprk-c-Icon--xl" />;
+
+export const rateShieldFilled = () => <SprkIcon iconName="rate-shield-filled" additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled" />;
+
+export const rateShieldFilledSmall = () => <SprkIcon iconName="rate-shield-filled-small" additionalClasses="sprk-c-Icon--filled" />;
+
+export const rates = () => <SprkIcon iconName="rates" additionalClasses="sprk-c-Icon--xl" />;
+
+export const recurringUpdate = () => <SprkIcon iconName="recurring-update" additionalClasses="sprk-c-Icon--xl" />;
+
+export const refinance = () => <SprkIcon iconName="refinance" additionalClasses="sprk-c-Icon--xl" />;
+
+export const reviews = () => <SprkIcon iconName="reviews" additionalClasses="sprk-c-Icon--xl" />;
+
+export const rocket = () => <SprkIcon iconName="rocket" additionalClasses="sprk-c-Icon--xl" />;
+
+export const rocketDocTwoColor = () => <SprkIcon iconName="rocket-doc-two-color" additionalClasses="sprk-c-Icon--xl" />;
+
+export const rotatePage = () => <SprkIcon iconName="rotate-page" additionalClasses="sprk-c-Icon--xl" />;
+
+export const sameDay = () => <SprkIcon iconName="same-day" additionalClasses="sprk-c-Icon--xl" />;
+
+export const savings = () => <SprkIcon iconName="savings" additionalClasses="sprk-c-Icon--xl" />;
+
+export const schools = () => <SprkIcon iconName="schools" additionalClasses="sprk-c-Icon--xl" />;
+
+export const search = () => <SprkIcon iconName="search" additionalClasses="sprk-c-Icon--xl" />;
+
+export const secureProcess = () => <SprkIcon iconName="secure-process" additionalClasses="sprk-c-Icon--xl" />;
+
+export const settings = () => <SprkIcon iconName="settings" additionalClasses="sprk-c-Icon--xl" />;
+
+export const settingsFilled = () => <SprkIcon iconName="settings-filled" additionalClasses="sprk-c-Icon--xl sprk-c-Icon--filled" />;
```

Comment:
A few of these go past the max-line length we have set in our eslintrc. I looked into this, and looks like we don't have any precommit hooks setup in this new docs site to stop the commit from happening if stuff like this occurs. Until we iron that out, we'll have to rely on our editors.